test/src/test/java/org/locationtech/geowave/test/landsat/CustomCRSLandsatIT.java

```diff
@@ -117,6 +118,10 @@ protected void processParameters(final OperationParams params) throws Exception
 
   @BeforeClass
   public static void startTimer() {
+
+    // GDAL version >= 2.3.0
+    System.loadLibrary("gdalalljni");
```

Comment:
I am curious as to why gdal needs to be explicitly loaded here.  Does that mean that third party developers need to explicitly load gdal when using those capabilities within GeoWave?

Reply:
nope, another left-over from fighting the tests, I'll delete